packages/core/getUserAvailability.ts

```diff
@@ -165,8 +166,9 @@ export async function getUserAvailability(
 
   const busyTimes = await getBusyTimes({
     credentials: user.credentials,
-    startTime: dateFrom.toISOString(),
-    endTime: dateTo.toISOString(),
+    // needed to correctly apply limits (weeks can be part of two months)
+    startTime: dateFrom.startOf("week").toISOString(),
+    endTime: dateTo.endOf("week").toISOString(),
```

Comment:
If this affects performance negatively, it would have to be pulled into both limit functions separately, or called with startOf/endOf only if any weekly limits are present.

Reply:
nice catch